qemu/tests/virtio_fs_multi_vms.py

```diff
@@ -0,0 +1,96 @@
+import logging
+import os
+
+from virttest import error_context
+from provider.virtiofs_utils import run_dd_test
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test to virtio-fs with the multiple VMs and virtiofs daemons.
+    Steps:
+        1. Create shared directories on the host.
+        2. Run virtiofs daemons on the host.
+        3. Boot guests on the host with virtiofs options.
+        4. Log into guest then mount the virtiofs targets.
+        5. Generate files on the mount points inside guests.
+        6. Compare the md5 between the host and the guest.
+        7. Compare the md5 among guests if multiple virtiofs
+           daemons share the source.
+
+    :param test: QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+    def create_dest_dir(vm, session, fs_dest):
+        error_context.context("%s: Create a destination directory %s "
+                              "inside guest." % (vm, fs_dest), logging.info)
+        session.cmd(cmd_create_dest_dir % fs_dest)
+
+    def mount_target(vm, session, fs_target, fs_dest):
```

Comment:
same here

Reply:
As you said, do we need consider to put:
```
        for fs in vm_params.objects('filesystems'):
            fs_params = vm_params.object_params(fs)
            fs_target = fs_params.get("fs_target")
            fs_dest = fs_params.get("fs_dest")
```
into `virtio_utils` ?  haha

Reply:
And like 
```
    vm = env.get_vm(params.get("main_vm"))
    vm.verify_alive()
    session = vm.wait_for_login()
```
they are all most used in each case, shoud we need put them into virtiofs_utils?

Reply:
> Hi, @sathnaga
> Please help me to review, thanks.
> cc @rajashreer7 @qunfang

@YongxueHong pls refer, https://github.com/autotest/tp-qemu/pull/2130/#issuecomment-635319993, tnx!